dependencies/tools/rstudio-tools.cmd

```diff
@@ -392,7 +392,7 @@ exit /b 0
 :add-vstools-to-path
 
 for %%Q in ("BuildTools" "Community") do (
-  for %%P in ("%ProgramFiles(x86)%" "%ProgramFiles%") do (
+  for %%P in ("%ProgramFiles%" "%ProgramFiles(x86)%") do (
```

Comment:
Switching this up since we now want to prefer build tools in C:/Program Files/ if it exists there.